### DISCUSSION TITLE
Recommended computer specs?

### DISCUSSION THREAD

**Discussion by melkalliny** [miniscope/minian#201] [Q&A]
The preprint describes 16GB RAM being enough to process "at least 20 minutes of recording data" - I'm wondering if there are any known issues with trying to process longer files using 16GB RAM?

**Comment by phildong** [comment]
In theory you can process quite arbitrary length of data with 16GB, especially if you're willing to increase `memory_limit`. 20min is what we have tested but we'd love to hear more feedback on this!
